client/login/wp-login/index.jsx

```diff
@@ -606,7 +567,7 @@ export class Login extends Component {
 									<HeadingSubText
 										isSocialFirst={ isSocialFirst }
 										twoFactorAuthType={ twoFactorAuthType }
-										isLostPassword={ isLostPassword }
```

Comment:
@raicem Following up on our chat https://github.com/Automattic/wp-calypso/pull/104104#discussion_r2161059772 - do you mind testing again with the Woo flows if this works? I think it works for me, but I may not be triggering the WooJPC flows correctly or something. 🙏

Reply:
Can confirm it works now for me, I wonder what I got wrong on Monday. Thank you for this @chriskmnds! 

I checked WooCommerce.com and WooJPC login and password reset. Their styles are consistent, so looks good! I'm happy to review it more thoroughly for other flows too, if needed.